spec/locales_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'spec_helper'
+
+describe Fastlane::Locales do
+  shared_examples 'from_xxx' do |key, fr_code, pt_code|
+    let(:method_sym) { "from_#{key}".to_sym }
+    it 'can find a locale from a single code' do
+      fr_locale = described_class.send(method_sym, fr_code)
+      expect(fr_locale).to be_instance_of(Fastlane::Locale)
+      expect(fr_locale.glotpress).to eq('fr')
+      expect(fr_locale.android).to eq('fr')
+      expect(fr_locale.google_play).to eq('fr-FR')
+    end
+
+    it 'can find locales from a multiple codes' do
+      locales = described_class.send(method_sym, [fr_code, pt_code])
+      expect(locales).to be_instance_of(Array)
+
+      expect(locales[0]).to be_instance_of(Fastlane::Locale)
+      expect(locales[0].glotpress).to eq('fr')
+
+      expect(locales[1]).to be_instance_of(Fastlane::Locale)
+      expect(locales[1].glotpress).to eq('pt-br')
+    end
+
+    it 'raises if one of the locale codes passed was not found' do
+      expect do
+        described_class.send(method_sym, [fr_code, 'invalidcode', 'pt-br'])
+      end.to raise_error(RuntimeError, "Unknown locale for #{key} code 'invalidcode'")
+    end
+  end
+
+  describe 'from_glotpress' do
+    include_examples 'from_xxx', :glotpress, 'fr', 'pt-br'
+  end
+
+  describe 'from_android' do
+    include_examples 'from_xxx', :android, 'fr', 'pt-rBR'
+  end
+
+  describe 'from_google_play' do
+    include_examples 'from_xxx', :google_play, 'fr-FR', 'pt-BR'
+  end
+
+  # @TODO: from_ios, from_app_store
```

Comment:
Add same tests for `from_ios` and `from_app_store` once [we complete the list](https://github.com/wordpress-mobile/release-toolkit/pull/296/files#r686194520)